src/pages/elements/pictograms/design.mdx

```diff
@@ -35,6 +35,17 @@ and work well in presentations and marketing communications.
 
   </ResourceCard>
 </Column>
+<Column colMd={4} colLg={4} noGutterSm>
+    <ResourceCard
+      subTitle="Pictograms Master"
+      aspectRatio="2:1"
+      href="https://github.com/carbon-design-system/carbon/tree/master/packages/pictograms/master"
```

Comment:
We can link directly to the file rather than making them go to github and click another link.
```suggestion
      href="https://github.com/carbon-design-system/carbon/raw/master/packages/pictograms/master/pictogram-master.ai"
```